src/helpers/optionalStacking/common.ts

```diff
@@ -16,3 +19,13 @@ export function getViewByTwoAttributes(
     }
     return [viewBy] as VisualizationObject.IVisualizationAttribute[];
 }
+
+export function verifyStackingConfig(props: IColumnChartProps | IBarChartProps): IChartConfig {
```

Comment:
The name of the function is a bit misleading. In my opinion, `verifyX` should return true/false meaning the X is valid or not. This function however _gets_ (extracts) something from `X` and _changes_ (sanitizes) it, so I would consider something like `extractSanitizedStackingConfig` or `getSanitizedStackingConfig`.

Reply:
it's done in fixup commit :)